packages/http-client-csharp/generator/TestProjects/CadlRanch/http/authentication/api-key/tspCodeModel.json

```diff
@@ -81,7 +81,7 @@
        "StatusCodes": [
         204
        ],
-       "BodyMediaType": "Json",
+       "BodyMediaType": "None",
```

Comment:
for http 204, apparently the media type should be none

Reply:
Should it be none, or omitted entirely?